Makefile

```diff
@@ -60,11 +58,16 @@ all: $(SPECIALRESOURCE)
 
 ##@ Development
 
-manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+go-install-binaries:
+	go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.6.1
+	go install golang.stackrox.io/kube-linter/cmd/kube-linter@v0.0.0-20210328011908-cb34f2cc447f
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.43.0
```

Comment:
You probably want that all of those to be in a separate script so you can call it from skipper as well I guess?
@yevgeny-shnaidman

Reply:
yes

Reply:
Fixed.